etc/plugins/TVirtualFitter/P030_TFitterMinuit.C

```diff
@@ -1,5 +1,5 @@
 void P030_TFitterMinuit()
 {
-   gPluginMgr->AddHandler("TVirtualFitter", "Minuit2", "TFitterMinuit",
-      "Minuit2", "TFitterMinuit(Int_t)");
+   //gPluginMgr->AddHandler("TVirtualFitter", "Minuit2", "TFitterMinuit",
```

Comment:
You can remove the whole file, and also the corresponding `TFitterFumili` plugin.

The implementations are long gone, removed in this commit:
465e8ee5589349e91a736e5f56bc9ff43780db2f

By the way, `math/minuit2/doc/index.txt` also has some comments about `TFitterMinuit` and `TFitterFumili` that should be removed.